src/bench/addrman.cpp

```diff
@@ -98,7 +98,7 @@ static void AddrManGetAddr(benchmark::Bench& bench)
     FillAddrMan(addrman);
 
     bench.run([&] {
-        const auto& addresses = addrman.GetAddr();
+        const auto& addresses = addrman.GetAddr(2500, 23);  // Compatibility with ADDRMAN_GETADDR_MAX_PCT and ADDRMAN_GETADDR_MAX
```

Comment:
This comment is confusing. I assume it's supposed to mean the original values thereof?

Reply:
Yes, so the values used in the bench stay the same. I've removed the comment since it's confusing.